### PR TITLE
chore(crowdin cli) Adding the crowdin sh scripts

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [BPO-4](https://bonitasoft.atlassian.net/browse/BPO-4)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'com.bonitasoft.gradle.bonita-release' version '0.1.46'
+    id 'com.bonitasoft.gradle.bonita-release' version '0.1.48'
```
Comment (tbouffard) [inline build.gradle]: ⚠️ unreleased version 😈

File: infrastructure/crowdin/pushTranslations.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+pull_request() {
+  PR="{\"title\": \"[${BRANCH_NAME}] Translations update\",\"body\": \"Latest translations made in [Crowdin](https://crowdin.com/project/bonita-bpm)\n!skipTests\", \"head\": \"feat/${BRANCH_NAME}/update-translations\", \"base\": \"${BRANCH_NAME}\"}"
+  echo "Create new pull request $PR"
+  curl -i -X POST -d "$PR" \
+     https://api.github.com/repos/bonitasoft/bonita-web-pages/pulls?access_token=$1
+}
+
+
+scriptFolder=`pwd`/$(dirname $0)
+. $scriptFolder/utils.sh
```
Comment (tbouffard) [inline infrastructure/crowdin/pushTranslations.sh]: ⚠️ the current script is not using functions declared in the `utils.sh` file
So I guess the `utils.sh` script should be removed

File: infrastructure/crowdin/pushTranslations.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#$3 github API Key
+pull_request() {
+  PR="{\"title\": \"[${BRANCH_NAME}] Translations update\",\"body\": \"Latest translations made in [Crowdin](https://crowdin.com/project/bonita-bpm)\n!skipTests\", \"head\": \"feat/${BRANCH_NAME}/update-translations\", \"base\": \"${BRANCH_NAME}\"}"
```
Comment (tbouffard) [inline infrastructure/crowdin/pushTranslations.sh]: ⚠️ url should be https://crowdin.com/project/bonita (bonita-bpm is a former name and redirect to bonita)

you should also removed `!skipTests` which is not used by the project jenkinsfile to skip tests when running a PR

**Comment by abirembaut** [comment]
No longer needed anymore since #272

**Pull request closed without merging**